### PR TITLE
Link a resource to its definition.

### PR DESCRIPTION
**Pull request by mirons-google**

### CODE REVIEW

File: _includes/_inventory/resources.md [hunk 1]
```diff
@@ -6,26 +6,23 @@ interested in isn't listed, please open an issue or contribute!
...
+| [Folders](https://cloud.google.com/resource-manager/reference/rest/v2beta1/folders) | Done | In Progress | TODO |
+| [GSuite Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-group) | Done | Done | TODO |
+| [GSuite Group Members](https://developers.google.com/admin-sdk/directory/v1/guides/manage-group-members) | Done | Done | TODO |
+| [IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) | In Progress | TODO | TODO |
+| [Load Balancer Forwarding Rules](https://cloud.google.com/compute/docs/reference/latest/forwardingRules#resource) | Done | In Progress | TODO |
+| [Organizations](https://cloud.google.com/resource-manager/reference/rest/v1beta1/organizations) | Done | N/A | TODO |
+| [Organizations Cloud IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) | Done | Done | TODO |
+| [Projects](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects) | Done | N/A | TODO |
+| [Projects Cloud IAM Policies](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getIamPolicy) | Done | Done | TODO |
```
Comment (blueandgold) [inline _includes/_inventory/resources.md]: Seems like Networks and Subnetworks are dropped?  Is that intentional?  Enforcer will need those.

Comment (mirons-google) [inline _includes/_inventory/resources.md]: Yeah, they were Open | Open | Open. I didn't see it useful to list them as there are lots of APIs we don't query.

Comment (blueandgold) [inline _includes/_inventory/resources.md]: Hmmm, I think those 2 items should stay as they do need to be done.  Would think that a roadmap is also for track things to be done, too.  :)

**Comment by mirons-google** [comment]
Agreed! But, let's create that elsewhere though.

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**